lib/parsers/binary.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
    * @param {*}      file.data      - The file contents. This will be whatever data type was returned by the resolver
    * @returns {Promise<Buffer>}
    */
-  parse (file) {
+  async parse (file) {
```

Comment:
is this required?

Reply:
In theory - it shouldn't. In practice, when all parsers do not return the promise or do not call the callback, everything blows up.
At least that was my observation - I'll take a look at it separately, and fix if needed.
For the time being, I can revert this particular change.